lib/inputstreamhelper/__init__.py

```diff
@@ -959,3 +996,23 @@ def info_dialog(self):
         text += localize(30830, url=config.ISSUE_URL)  # Report issues
 
         Dialog().textviewer(localize(30901), text)
+
+    def rollback_libwv(self):
+        """Backup current lib and rollback to a version specified by the user"""
+        backup_path = self._backup_path()
+        versions = os.listdir(backup_path)
+
+        if 'x86' in self._arch():
+            show_versions = versions
+        else:
+            show_versions = []
+
+            for version in versions:
+                lib_version = self._get_lib_version(os.path.join(backup_path, version, config.WIDEVINE_CDM_FILENAME[system_os()]))
+                show_versions.append('{}    ({})'.format(lib_version, version))
+
+        version = Dialog().select(localize(30057), show_versions)
```

Comment:
Showing an empty `Dialog.select()` when the backup dir is empty is not nice. I think it's better to show a `Dialog().notification` with an error message to the user.

Reply:
Rollback would probably only ever be used if it was installed at least once, so it would at least show the current version, but just to be safe I added a check.